scripts/files/terraform-config.tfrc

```diff
@@ -0,0 +1,5 @@
+provider_installation {
+  filesystem_mirror {
+    path = "/opt/terraform/plugins"
+  }
+}
```

Comment:
Will Coder attempt to download from the public registry if there isn't something in this mirror, like our other cloud examples? I genuinely don't know.

Reply:
I'm not sure either; so I'm going to omit this change.